components/NewRealmWizard/components/steps/AddNFTCollectionForm.tsx

```diff
@@ -59,11 +71,16 @@ async function enrichItemInfo(item, uri) {
   }
 }
 
-async function enrichCollectionInfo(connection, collectionKey) {
-  const data = await deprecated.Metadata.findByMint(connection, collectionKey)
-
-  const collectionData = data!.data!.data
+async function enrichCollectionInfo(
+  connection: Connection,
+  collectionKey: string
```

Comment:
May be interesting to use `collectionKey: PublicKey` here.

We use `toBase58()` on PublicKey to then re-transform the string to PublicKey

The actual call:
```
const collectionInfo = await enrichCollectionInfo(
      connection,
      metaplexData.collection.address.toBase58()
    )
```

Reply:
This is one thing that i will left because in whole component it is mostly used as string for view or operations and whole component has most of things as any... so when i change it to pubkey it will break things that im not aware of.  It would be good to rewrite a bit the logic in there and add normal typing becuase in some places it seems weird with new metaplex but i don't want to do it now.